docs/components/css/colors.css

```diff
@@ -9,6 +9,7 @@
     --color-dark-blue: #032f3b;
     --color-dark-grey: darkgrey;
     --color-white: white;
+    --color-orange:orange;
```

Comment:
Maybe using a toned down version or orange would be better here .

Reply:
sure